### PR TITLE
Update test readme and results 

### PR DESCRIPTION
**Pull request by antejavor**
- [x] Update results 
- [x] Update README

### CODE REVIEW

File: import/README.md [hunk 1]
```diff
@@ -1,26 +1,48 @@
 # Import
 
-In this directory you'll find the best strategies to import CSV and Cypher queries into Memgraph.
+In this directory, you'll find the best strategies for importing CSV and Cypher queries into Memgraph.
 
+The cypher and csv folders contain examples of how to load a CYPHERL file and LOAD CSV via pymgclient and neo4jpython client.
```
Comment (katarinasupe) [inline import/README.md:5]: link the folders +
```suggestion
The cypher and csv folders contain examples of how to load a CYPHERL file and run LOAD CSV via `pymgclient` and `neo4j` Python clients. 
```

File: import/README.md [hunk 2]
```diff
@@ -1,26 +1,48 @@
 # Import
 
-In this directory you'll find the best strategies to import CSV and Cypher queries into Memgraph.
+In this directory, you'll find the best strategies for importing CSV and Cypher queries into Memgraph.
 
+The cypher and csv folders contain examples of how to load a CYPHERL file and LOAD CSV via pymgclient and neo4jpython client. 
 
-## Node import speed reference
+Before running any import script, make sure the `../dataset/graph500/download_dataset.sh` is executed first,
```
Comment (katarinasupe) [inline import/README.md:7]: Link to the script or instructions on how to run it

File: import/README.md [hunk 3]
```diff
@@ -1,26 +1,48 @@
...
+Before running any import script, make sure the `../dataset/graph500/download_dataset.sh` is executed first, 
+It will pull all necessary data into the dataset directory needed to run the tests. 
+
+If you are running a test, for example, the node import process in pymgclient, you need to pass the dataset size: 
+
+```bash
+python3 ./cypher/pymgclient/concurrent_node_import.py small
+```
+
+For concurrent LOAD CSV, Memgraph needs to be in `IN_MEMORY_ANALYTICAL` mode.
```
Comment (katarinasupe) [inline import/README.md:16]: link to docs (in memory analytical)

File: import/README.md [hunk 4]
```diff
@@ -1,26 +1,48 @@
...
+For concurrent LOAD CSV, Memgraph needs to be in `IN_MEMORY_ANALYTICAL` mode. 
+
+## Test reference 
+
+Below are the numbers representing import speed in different scenarios. The tests were run on the following hardware: 
+
+CPU: AMD Ryzen 5 2600 Six-Core Processor
+RAM: 2x8GB 2133 MT/s DDR4
+
+All tests used ten concurrent database connections. All tests were made on docker containers; native versions of Memgraph will yield even better results.
```
Comment (katarinasupe) [inline import/README.md:25]:
```suggestion
All tests used ten concurrent database connections. All tests were made on Docker containers; native versions of Memgraph will yield even better results.
```

File: import/README.md [hunk 5]
```diff
@@ -1,26 +1,48 @@
...
+
+### Node import speed reference
 
 Nodes per second = n/s
 Peak memory pressure in megabytes = MB 
 
-TEST 1: 
+TEST 1: neo4jpython
```
Comment (katarinasupe) [inline import/README.md:32]: isn't the driver called just neo4j?

File: import/README.md [hunk 6]
```diff
@@ -31,17 +53,18 @@ Peak memory pressure in megabytes = MB
 
 Not possible = np
 
-TEST 1: 
+TEST 1: neo4jpython
```
Comment (katarinasupe) [inline import/README.md:56]: isn't the driver called just neo4j?

File: import/README.md [hunk 7]
```diff
@@ -31,17 +53,18 @@ Peak memory pressure in megabytes = MB
 
 Not possible = np
```
Comment (katarinasupe) [inline import/README.md:54]: Is it better to go with classic notation [N/A](https://en.wikipedia.org/wiki/N/A)?

**Review by katarinasupe — CHANGES_REQUESTED** [review]

File: import/README.md (hunk 2 above)
Comment (antejavor) [reply import/README.md:7]: Added

File: import/README.md (hunk 3 above)
Comment (antejavor) [reply import/README.md:16]: Added.

File: import/README.md (hunk 5 above)
Comment (antejavor) [reply import/README.md:32]: Yep, which I find a bit confusing but, will rename to just neo4j.

File: import/README.md (hunk 7 above)
Comment (antejavor) [reply import/README.md:54]: Yeah, great suggestion.

**Review by matea16 — APPROVED** [review]

**Review by katarinasupe — APPROVED** [review]

**Pull request merged**